### DISCUSSION TITLE
Opened programs not showing in taskbar on second monitor

### DISCUSSION THREAD

**Discussion by IJuanTM** [valinet/ExplorerPatcher#1423] [Q&A]
Hi,

I have the following option turned on, but it doesn't work reliably with Explorer Patcher installed:
![image](https://user-images.githubusercontent.com/43535731/236330258-26a822e7-dbc0-405d-b2d4-8286feb73d58.png)

I am on Windows 11 Pro 22H2 (OS Build 23440.1000).

Does anyone have the same or a similar issue? And does anyone maybe know a fix for this? Would be greatly appreciated! 

Thanks!

**Comment by vippium** [comment]
EP is not intended for Insider builds...only stable builds bro...
